workflow.rst

```diff
@@ -334,6 +334,10 @@ order:
     * ``workflow.[workflow name].announce``
     * ``workflow.[workflow name].announce.[transition name]``
 
+    You can avoid this event by using the context::
```

Comment:
```suggestion
    You can avoid those events by using the context::
```

according to the above sentence:

> The three events being dispatched are:

wdyt?

Reply:
Thanks